tests/ui-testing/pages/createOrgPage.js

```
@@ -0,0 +1,42 @@
+        const fetchResponse = await fetch(
+            `${process.env.INGESTION_URL}/api/organizations`,
+            {
+                method: "POST",
+                headers: headers,
+                body: JSON.stringify(newOrgBody),
+            }
+        );
+        try {
+            const response = await fetchResponse;
+            const contentType = response.headers.get("content-type");
+            if (contentType && contentType.includes("application/json")) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)